README.md

```diff
@@ -9,9 +9,23 @@ npm
 I recommend using nvm
 
 ## Installation
+### Packages
 ```
 npm install
 ```
+### Database
+To create database run:
+```
+psql -c "CREATE DATABASE snippethero_development WITH ENCODING 'UTF8'
```

Comment:
@Vakrim You miss```"``` at the end.
also I had some problems with creation of the database with that command and finally ended with ```createdb -E UTF8 snippethero_development```